src/__test__/components/plots-and-tables/markerHeatmap.test.jsx

```diff
@@ -9,28 +9,28 @@ import _ from 'lodash';
 import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
 import {
   initialPlotConfigStates,
-} from '../../../redux/reducers/componentConfig/initialState';
+} from 'redux/reducers/componentConfig/initialState';
 
-import { loadGeneExpression } from '../../../redux/actions/genes/index';
-import * as markerGenesLoaded from '../../../redux/reducers/genes/markerGenesLoaded';
-import * as configUpdated from '../../../redux/reducers/componentConfig/updateConfig';
-import initialExperimentState from '../../../redux/reducers/experimentSettings/initialState';
-import rootReducer from '../../../redux/reducers/index';
-import * as loadConfig from '../../../redux/reducers/componentConfig/loadConfig';
-import MarkerHeatmap from '../../../pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index';
-import * as cellSetsLoaded from '../../../redux/actions/cellSets/loadCellSets';
-import * as loadedProcessingConfig from '../../../redux/actions/experimentSettings/processingConfig/loadProcessingSettings';
+import { loadGeneExpression } from 'redux/actions/genes/index';
+import * as markerGenesLoaded from 'redux/reducers/genes/markerGenesLoaded';
+import * as configUpdated from 'redux/reducers/componentConfig/updateConfig';
+import initialExperimentState from 'redux/reducers/experimentSettings/initialState';
+import rootReducer from 'redux/reducers/index';
+import * as loadConfig from 'redux/reducers/componentConfig/loadConfig';
+import MarkerHeatmap from 'pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index';
+import * as cellSetsLoaded from 'redux/actions/cellSets/loadCellSets';
+import * as loadedProcessingConfig from 'redux/actions/experimentSettings/processingConfig/loadProcessingSettings';
 
 enableFetchMocks();
 jest.mock('localforage');
-jest.mock('../../../components/plots/Header', () => () => <div />);
-jest.mock('../../../utils/socketConnection', () => ({
+jest.mock('components/plots/Header', () => () => <div />);
+jest.mock('utils/socketConnection', () => ({
   __esModule: true,
   default: new Promise((resolve) => {
     resolve({ emit: jest.fn(), on: jest.fn(), id: '5678' });
   }),
 }));
-jest.mock('../../../utils/work/fetchWork', () => ({
+jest.mock('utils/work/fetchWork', () => ({
```

Comment:
the actual `ListGenes` response contains more stuff like `"{"total": 16031, "rows": [{"gene_names": "S100A9", "dispersions": 23.4435, "_row": "ENSG00000163220"},`. Unless there is a specific reason not include them I would add them for the sake of completeness. At least the `total` field.

Reply:
for the `GeneExpression`, I was not able to find the `rawExpression`, the response looks like `"{"S100A8": {"truncatedExpression": {"min": 0.0, "max": 5.7651, "expression": [0, null, null, 0, 0, 0, 0, 5.7651, null, 0, 0, null, 1.4358, 0, 0, null, 0, null, 1.2752, 5.4748, null, 0, null, null, 4.5487, null, 0, 0, null, 0, null...`. Verify whether this mock is outdated and update it if needed.

Reply:
List genes is not used, so I'm removing it.